### PR TITLE
Updating Env nci-crdc.datacommons.io - 2022.09 1664556619

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2022.09 to nci-crdc.datacommons.io

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- metadata
  - Database migration for MDS (uc-cdis/metadata-service#63)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (uc-cdis/indexd#336)
- portal
  - Replaced loading  the data in  `iframes` in Workspace page with Redirect component. (uc-cdis/data-portal#1067)
  - Deprecating the use of `default-workspaces` i. e an individual commons will no longer be able to deploy standalone jupyterhub images, they may use [hatchery service](https://github.com/uc-cdis/hatchery) instead. (uc-cdis/data-portal#1067)
  - `studyRegistrationConfig` in `gitops.json` must now be placed within the `regsitrationConfigs.features` block [[Refer this PR](https://github.com/uc-cdis/cdis-manifest/pull/5095)] (uc-cdis/data-portal#1067)

File: nci-crdc.datacommons.io/manifest.json [hunk 1]
```diff
@@ -7,23 +7,23 @@
...
-    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2022.08",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2022.09",
+    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2022.09",
     "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
-    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2022.08",
-    "datareplicate": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/dcf-dataservice:fix_update_redaction",
-    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:6.1.1",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2022.09",
+    "datareplicate": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/dcf-dataservice:2022.09",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2022.09",
```
Comment (trevars) [inline nci-crdc.datacommons.io/manifest.json:15]: I believe we still want fence:6.1.1. That's what we have in staging.

**Review by trevars — CHANGES_REQUESTED** [review]

**Review by trevars — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- metadata
  - Database migration for MDS (uc-cdis/metadata-service#63)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (uc-cdis/indexd#336)
- portal
  - Replaced loading  the data in  `iframes` in Workspace page with Redirect component. (uc-cdis/data-portal#1067)
  - Deprecating the use of `default-workspaces` i. e an individual commons will no longer be able to deploy standalone jupyterhub images, they may use [hatchery service](https://github.com/uc-cdis/hatchery) instead. (uc-cdis/data-portal#1067)
  - `studyRegistrationConfig` in `gitops.json` must now be placed within the `regsitrationConfigs.features` block [[Refer this PR](https://github.com/uc-cdis/cdis-manifest/pull/5095)] (uc-cdis/data-portal#1067)

**Comment by PlanXCyborg** [comment]
# nci-crdc.datacommons.io/manifest.json
## Deployment changes
- metadata
  - Database migration for MDS (uc-cdis/metadata-service#63)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (uc-cdis/indexd#336)
- portal
  - Replaced loading  the data in  `iframes` in Workspace page with Redirect component. (uc-cdis/data-portal#1067)
  - Deprecating the use of `default-workspaces` i. e an individual commons will no longer be able to deploy standalone jupyterhub images, they may use [hatchery service](https://github.com/uc-cdis/hatchery) instead. (uc-cdis/data-portal#1067)
  - `studyRegistrationConfig` in `gitops.json` must now be placed within the `regsitrationConfigs.features` block [[Refer this PR](https://github.com/uc-cdis/cdis-manifest/pull/5095)] (uc-cdis/data-portal#1067)

**Pull request merged**